.elixir_ls/build/test/lib/litmus/ebin/litmus.app

```diff
@@ -1,6 +1,6 @@
 {application,litmus,
```

Comment:
Can you remove these `.elixir_ls/*` files as well since we're adding it to the gitignore?